plugins/techdocs/src/home/components/OwnedContent.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useCopyToClipboard } from 'react-use';
+import { generatePath } from 'react-router-dom';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import ShareIcon from '@material-ui/icons/Share';
+import {
+  Content,
+  ContentHeader,
+  SupportButton,
+  Table,
+  identityApiRef,
+  useApi,
+  EmptyState,
+  Button,
+  SubvalueCell,
+  Link,
+} from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+
+import { rootDocsRouteRef } from '../../plugin';
+
+export const OwnedContent = ({ value }: { value: Entity[] | undefined }) => {
+  const [, copyToClipboard] = useCopyToClipboard();
+  const identityApi = useApi(identityApiRef);
+  const userId = identityApi.getUserId();
+
+  if (!value || !userId) return null;
+
+  const ownedDocuments = value
+    .filter((entity: Entity) => entity?.spec?.owner === userId)
```

Comment:
Would be cool to also see documents that are owned via group membership. `plugin-catalog-react` has a [`isOwnerOf`](https://github.com/backstage/backstage/blob/2cc2d1ad1788d5680d77b6192e40870d14a83f0c/plugins/catalog-react/src/utils/isOwnerOf.ts#L29) helper for this which is also used in the catalog. However, the code to find the user entity is is not directly usable here and also not very complete since it waits for the new user-identity mapping ([`useOwnUser`](https://github.com/backstage/backstage/blob/2cc2d1ad1788d5680d77b6192e40870d14a83f0c/plugins/catalog/src/components/useOwnUser.ts#L26)).

Reply:
@dhenneke Very good point. Thanks for pointing that out. I'll see what the status is of `useOwnUser` otherwise I might be able to use some workaround here until its ready!

Reply:
> Would be cool to also see documents that are owned via group membership. `plugin-catalog-react` has a [`isOwnerOf`](https://github.com/backstage/backstage/blob/2cc2d1ad1788d5680d77b6192e40870d14a83f0c/plugins/catalog-react/src/utils/isOwnerOf.ts#L29) helper for this which is also used in the catalog. However, the code to find the user entity is is not directly usable here and also not very complete since it waits for the new user-identity mapping ([`useOwnUser`](https://github.com/backstage/backstage/blob/2cc2d1ad1788d5680d77b6192e40870d14a83f0c/plugins/catalog/src/components/useOwnUser.ts#L26)).

Updated to use this but @backstage/maintainers probably want to take a look if its ok to use or if we should wait. I exported the hook `useOwnUser` from the catalog plugin to be able to use it, let me know if I should change back again. 
<img width="1440" alt="Screen Shot 2021-03-04 at 1 11 37 PM" src="https://user-images.githubusercontent.com/25153114/109962892-0997c700-7cec-11eb-9f5d-84a576d9221b.png">